assets/grayscale/app/controllers/grayscales_controller.rb

```diff
@@ -0,0 +1,5 @@
+class GrayscalesController < ApplicationController
+  layout "grayscale"
+	def index
```

Comment:
Looks like the indentation is a bit off here.  Make sure you're using spaces and not tabs :smile: